model/logs/mobile-crashes.yaml

```diff
@@ -0,0 +1,47 @@
+groups:
+  - id: mobile-crash-event
+    prefix: hello
+    type: event
+    brief: >
+      This document defines attributes for crashes represented using Log Record Events.
+      `event.domain` is 'device' and `event.name` is 'crash'.
+    attributes:
+      - ref: ios.state
+        requirement_level:
+          recommended: If the crash occurred on an iOS device.
+      - ref: android.state
+        requirement_level:
+          recommended: If the crash occurred on an Android device.
+      - ref: exception.stacktrace
+        requirement_level:
+          recommended: If the crash occurred on an Android device.
+        brief: >
+          The contents of printStackTrace() being called on the throwable
+      - ref: thread.id
+        requirement_level:
+          recommended: If the crash occurred on an Android device.
+        brief: >
+          The id of the thread as output by Thread.currentThread().getId()
+      - ref: thread.name
+        requirement_level:
+          recommended: If the crash occurred on an Android device.
+        brief: >
+          The name of the thread as output by Thread.currentThread().getName()
+      - ref: exception.message
+        requirement_level:
+          recommended: If the crash occurred on an Android device.
+      - id: ios.crash_report
+        requirement_level:
+          recommended: If the crash occurred on an iOS device.
+        brief: >
+          The contents from PLCrashReporter
```

Comment:
Would be possible to make the reporter agnostic?
In case I use [KSCrash](https://github.com/kstenerud/KSCrash), I'd like to use the very same data model.
Maybe introducing a new property that tells what's the reporter format, such as `KSCrash`, `PLCrashReporter` or anything else.